plugins/hydra_rq_launcher/tests/test_rq_launcher.py

```diff
@@ -48,13 +48,14 @@ class TestRQLauncherIntegration(IntegrationTestSuite):
 
 # https://github.com/rq/rq/issues/1244
 @mark.filterwarnings("ignore::DeprecationWarning")
-def test_example_app(hydra_sweep_runner: TSweepRunner) -> None:
+@mark.parametrize("config_name", ["config", "config_redis_ssl"])
```

Comment:
instead of parameterizing on the config, you can parameterize on additional overrides.

```python
@mark.parametrize("overrides", [[], ["hydra.launcher.redis.ssl=true]])
```

And then add the overrides to those passed to hydra_sweep_runner.

Reply:
Test was changed, see f8780eaf765128aa5e5795868554a34ae19670a6.